src/text/writer.rs

```diff
@@ -364,9 +410,11 @@ mod tests {
     }
 
     #[test]
-    #[ignore]
     fn write_clob() {
-        todo!();
+        writer_test(
+            |w| w.write_clob(&[b'a', b'"', b'\'', b'\n']),
+            "{{\"a\\\"'\\n\"}}\n",
```

Comment:
I know the other unit tests in this module are pretty sparse, but would you mind adding a few more `writer_test`s to this one? Try an ascii string, an ascii string with special escape characters, some emoji, etc.